Update Pond Strider, Flea, associated outfits (#4494)
endless-sky/endless-sky: data/hai/hai missions.txt

mission "Hai Solifuge response"
	landing
	invisible
	to offer
		has "Wanderers Solifuge Recon 2: done"
	on offer
		event "Hai Pond Striders released" 10
		fail

event "Hai Pond Striders released"
	fleet "Large Hai"
		add variant 1
			"Hai Pond Strider" 2
			"Flea (Pulse)" 8
			"Flea" 8
			"Lightning Bug" 2
		add variant 1
			"Hai Pond Strider" 3
			"Flea (Pulse)" 12
			"Flea" 12
		add variant 1
			"Hai Pond Strider"
			"Flea" 8
			"Geocoris"
			"Aphid (Anti-Missile)" 2
			"Grasshopper"
		add variant 1
			"Hai Pond Strider" 2
			"Flea (Pulse)" 8
			"Flea" 8
			"Centipede"

mission "Strider: View of the Pond"
	description "Deliver a message regarding new Unfettered ships from Ambassador Sayari to Hai elders on <destination>."
	name "Message Hai-home"
	landing
	source "Vara K'chrai"
	destination "Hai-home"
	to offer
		has "Wanderers Solifuge Recon 2: done"
	on offer
		conversation

> 			`"Ambassador Sayari has a message for the Hai elders about the Solifuge, and needs your help to deliver it to <destination>."`

Reviewer: She does? Isn't she a Hai? Why would she need your help? This message is kinda too short and could bear expansion.

				accept

mission "Strider: Have language, will travel"
	description "Take Ambassador Yashili and her delegation to <destination>."
	name "Have language, will travel"
	landing
	source "Hai-home"
	destination "Greenwater"
	passengers 3

> 	blocked "The Hai wish to send a delegation to a species that has hull repair technology, and will require 3 bunks for the trip."

Reviewer: ```suggestion
	blocked "The Hai wish to send a delegation to a species that has hull repair technology, and will require <bunks> bunks for the trip."
```

	to offer
		has "Strider: View of the Pond: done"
	on offer
		log `The Hai are working on a medium warship that carries drones in response to the Unfettered, but need help contacting a species with hull repair technology to make it viable.`
		conversation
			`As you land the <ship>, you send word that you need to deliver your message from Ambassador Sayari to the Hai elders.  An appointment is arranged, and within hours, you find yourself before them once again.`

> 			`The elders tell you how the Solifuge was not entirely a surprise to the Hai government, but that it posed a threat to the Hai public mindset that couldn't be ignored.`

Reviewer: Starting with even just a bit of dialogue rather than a summary of it would be better IMO.

> 			`They continue ""A small, short range drone carrier and combat drone solution is being worked on, but initial simulations predict that actual escort casualties would go up unless significant hull repair technology could be implemented to fix docked drones that would otherwise weigh down the agile warship."`

Reviewer: This is a run-on sentence that needs to be broken up.

					goto neither
			label both
			`You consider that both the Remnants and the Coalition have technology that repairs hulls.`
			choice
				`(Mention the Remnants, and offer to take a delegation to Aventine.)`
					goto aventine
				`(Mention the Coalition, and offer to take a delegation to the Ring of Friendship.)`
					goto ring
				`(The Hai are not ready to learn of hull repair technology yet.)`
					goto neither
			label ring
			apply
				set "Strider: Coalition"
			`The grateful Hai ambassadors request a stop at <destination> before proceeding to the Ring of Friendship, so that they can pick up the latest version of the translation device they are using to communicate with you.`
				accept
			label aventine
			apply
				set "Strider: Remnants"
			`The grateful Hai ambassadors request a stop at <destination> before proceeding to Aventine, so that they can pick up the latest version of the translation device they are using to communicate with you.`
				accept
			label neither
			`You explain that you have not encountered a friendly civilization with hull repair technology, but that when you do, you might be able to introduce the Hai to them.`
				defer
	on visit

> 		dialog `It appears that the delegation is bunked on escorts that have not arrived in this system yet.  Take off and wait for them.`

Reviewer: ```suggestion
		dialog `It appears that some members of the delegation are being transported by escorts that have not arrived in this system yet.  Take off and wait for them.`
```

mission "Strider: A tale of two remnants"
	description "Transport Ambassador Yashili and her delegation to <destination>."
	name "A tale of two remnants"
	landing
	source "Greenwater"
	destination "Aventine"
	passengers 3

> 	blocked "The Hai are sending a delegation to <destination>, and will require 3 bunks for the trip."

Reviewer: ```suggestion
	blocked "The Hai are sending a delegation to <destination>, and will require <bunks> bunks for the trip."
```

	to offer
		has "Strider: Have language, will travel: done"
		has "Strider: Remnants"
	on offer
		conversation
			`One of the Hai ambassadors rushes out of the <ship> as soon as you touch down, and returns minutes later with a set of translation devices similar to the one you acquired to speak with the Wanderers.  Ambassador Yashili urges "Now to <destination>, so we may meet the Remnants."`
				accept
	on complete
		log `Helped establish a Hai embassy on Aventine.`
	on visit

> 		dialog `It appears that the delegation is bunked on escorts that have not arrived in this system yet.  Take off and wait for them.`

Reviewer: ```suggestion
		dialog `It appears that some members of the delegation are being transported by escorts that have not arrived in this system yet.  Take off and wait for them.`
```

mission "Strider: Widen the new coalition"
	description "Transport Ambassador Yashili and her delegation to <destination>."
	name "Widen the new coalition"
	landing
	source "Greenwater"
	destination "Ring of Friendship"
	passengers 3
	blocked "The Hai are sending a delegation to <destination>, and will require 3 bunks for the trip."
	to offer
		has "Strider: Have language, will travel: done"
		has "Strider: Coalition"
	on offer
		conversation
			`One of the Hai ambassadors rushes out of the <ship> as soon as you touch down, and returns minutes later with a set of translation devices similar to the one you acquired to speak with the Wanderers.  Ambassador Yashili urges "Now to <destination>, so we may meet the Heliarchs."`
				accept
	on complete
		log `Helped establish a Hai embassy on the Ring of Friendship.`
	on visit
		dialog `It appears that the delegation is bunked on escorts that have not arrived in this system yet.  Take off and wait for them.`

mission "Strider: Of the Pond R"
	description "Transport an ambassadorial entourage to <destination>."
	name "Establish Remnant embassy"
	landing
	source "Aventine"
	destination "Hai-home"
	passengers 3

> 	blocked "The Remnants are sending a delegation to <destination>, and require 3 bunks for the trip."

Reviewer: ```suggestion
	blocked "The Remnants are sending a delegation to <destination>, and require <bunks> bunks for the trip."
```

	to offer
		has "Strider: A tale of two remnants: done"
	on offer
		conversation

> 			`A Remnant prefect immediately greets the Hai delegation, and ushers them away to the Spaceport. Three Remnants approach you and express that they are interested in immersing themselves in Hai culture: request to return to <destination> with you.`

Reviewer: ```suggestion
			`A Remnant prefect immediately greets the Hai delegation, and ushers them away to the Spaceport. Three Remnants approach you and express that they are interested in immersing themselves in Hai culture and request to return to <destination> with you.`
```

			goto remnant
		`	"There are a group of three alien species far from here known as the Coalition that have outfits able to repair a ship in flight."`
			goto coalition
	label knowRemnant
	choice
		`	"A group of humans in an area of space known as the Ember Waste have ships that are able to repair themselves."`
			goto remnant
	label knowCoalition
	choice
		`	"There are a group of three alien species far from here known as the Coalition that have outfits able to repair a ship in flight."`
			goto coalition
			
	label remnant
	apply
		set "strider: remnant"
	``
		goto end
	label coalition
	apply
		set "strider: coalition"
	
	label end
	`	"Wonderful!" Torsa says. "This should turn many years of working into only a few months to finish our ship."`
	`	The elders and Torsa continue to discuss for a short while longer, deciding that Torsa and his team should create a small number of new ships to defend Cloudfire while you travel to acquire hull repair technology to finish the ship. "We will put together a delegation for you to transport to these people you mention. They will be waiting in the spaceport shortly."`
		decline

> mission "Strider: Remnant Hull Repair"

Reviewer 2: I'm not entirely a fan of this mission. I understand why it is here, and it works. 

That being said, I'd much rather have an informational Remnant mission that the player can encounter regardless of any Hai missions, rather than have such a piece of info hidden away to only come up if the player triggers the Strider mission set first.

			`	"We too have encountered these Korath," Yashili says in response. "Although, we had not taken their technology. This may be of great use to us."`
			`	The Remnant agree to meet in the spaceport with a copy of the research done on the Korath technology, as well as a collection of the small robots for them to study.`

mission "Strider 3 (Remnant)"
	name "Technology Obtained"
	description "The delegation has successfully obtained hull repair technology from the Remnant. Return to <destination> so that the Hai can begin developing this technology for their new ship."
	source "Aventine"
	destination "Hai-home"
	passengers 3
	cargo "repair technology" 12
	blocked "You need <capacity> in order to return the Hai delegation and the repair technology back to <planet>."
	to offer
		has "Strider 3 (Remnant): done"
	on offer
		conversation
			`The Remnant hand the Hai a data chip containing a copy of all the information that they have on the Korath technology and load a crate full of samples of the Korath outfits onto your ship. Yashili thanks the Remnant for their generosity and boards your ship, ready to return to <destination>.`
				accept

> 

Reviewer 2: I'm not sure whether my PR directly to the branch or a suggestion here is easier, so I have done both. This one only has the Remnant missions themselves, not the change to the initial requirements. (In my PR I have also changed it from requiring the blood test to requiring the license. 
a) Makes it slightly more likely the coalition side actually comes up, 
b) Means the player has actually earned some trust to be trusted with arranging this thing, and 
c) move the mission from something going on while the player is just trying to get the Remnant's trust to something going on while they are learning about their character.

```suggestion

mission "Strider 2 (Remnant)"
	landing
	name "Contact the Remnant"
	description "Take a message from the Hai to <destination> to set up a meeting between the Remnant and the Hai."
	source "Hai-home"
	stopover
		government "Remnant"
	to offer
		has "strider: remnant"
	on offer
		conversation
			`You enter the spaceport and find a Hai delegation ready to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I wasn't sure how far we will be traveling, so I packed assuming we will be traveling a great distance across the galaxy."`
			choice
				`	"Wait a minute! This group doesn't react well to strangers. How about I setup a meeting first?"`
				`	"This is a bad idea. Sorry, but I am backing out."`
					decline
			`	The Hai looking at each other and chatter quietly among themselves, then Ambassador Yashili speaks up. "You know this group, and how to make the best impression on them." She pauses to whisper to another Hai, who runs off. The ambassador turns back to you. "I will provide you with a letter with a brief introduction to us, an explanation of why we would like to meet them, and an invitation to establish diplomatic relations."`
			`	Moments later, the second Hai runs back into the spaceport carrying a sealed envelope and hands it to the Ambassador. She turns to you. "Here it is. Please take this to these reclusive humans for us."`
				accept
	on stopover
		conversation
			`Looking around the spaceport, you are not sure who to talk to about such a request. Eventually you go to the starport information desk to ask them. After a few minutes they tell you that someone will meet with you, and invite you to wait in the cafeteria.`
			`	After an hour of watching Remnant eating and gesturing to each other, an androgynous Remnant approaches your table. You try to decide what age they are, but while they do not seem young, neither do they seem particularly old.`
			branch signlanguage
				has "Remnant: Learn Sign Follow Up: offered"
			`	"Greetings, Captain <first>." The stranger chants. "My name is Rayn, and my colleague said you had a request from another faction?"`
				goto stridercontinue
			label signlanguage
			`	"Greetings, Captain <first>." The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction?"`
			label stridercontinue
			choice
				`	"The Hai are hoping to learn from someone."`
				`	"The Hai need help defending their space."`
			`	"So the Hai want to improve their military technology? I presume this is in response to their long-standing war against the Unfettered?"`
			choice
				`	"Yes, although they don't think of it as such."`
				`	"It isn't a war, it is a defensive action."`
			`	Rayn looks amused. "It does not matter what they call it, a war by any name is just as bloody. But that aside, how do they know about us to make such a request?"`
			choice
				`	"They asked if I knew anyone who had in-flight repair technology, and I said I knew a group of human researchers."`
				`	"They were looking for someone who had repair technology, and I said I could find someone."`
			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes his eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these co-ordinates on Deadman's Cove, in the Almach system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`
	on complete
		conversation
			`Shortly after landing on <destination> you spot Ambassador Yashili's aide rushing across the tarmac to meet you. "Please tell me they agreed? They did, right?" You barely have the chance to nod before the aide continues on. "The Ambassador will be ready in just a moment. She will meet you in the starport shortly. Thank you!" And with that, the officious Hai scurries off back the way it came.`

mission "Strider 3 (Remnant)"
	name "Contact the Remnant"
	description "Transport Ambassador Yashili and her delegation to <destination> to attempt to obtain hull repair technology from the Remnant."
	source "Hai-home"
	destination "Deadman's Cove"
	infiltrating
	passengers 3
	blocked "The Hai are sending a delegation to <destination>, and will require 3 bunks for the trip."
	to offer
		has "strider: remnant 2 (Remnant)"
	on offer
		conversation
			`You enter the spaceport and find the Hai delegation meant to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I heard that we will be traveling a great distance across the galaxy. May we have a safe journey."`
			`	"Is there anything that we should know about these Remnant before meeting them?" another member of the delegation asks.`
			choice
				`	"The Remnant are a group of humans, so there isn't anything special you should need to know."`
					goto normal
				`	"The Remnant communicate with each other using sign language, but they are still capable of communicating with us through speech."`
			`	"That is quite interesting," Yashili says with nods of agreement from the other two members of the delegation. "I find it odd that a group capable of speech would prefer to communicate with one another using sign language. This may be a more exciting journey than I expected."`
			label normal
			`	The delegation gathers their things and boards your ship, and you plot a course for <destination>. Hopefully the Remnant will be willing to share with an alien species they just met.`
				accept
	on visit
		dialog `It appears that the delegation is bunked on escorts that have not arrived in this system yet. Take off and wait for them.`
	on complete
		conversation
			`At the designated co-ordinates there is a small encampment set up, looking vaguelly reminiscent of desert traders from old movies. You note there are even a few camels tied near a water trough. When you touch down next to it, the delegation steps out with a confused look. "Captain, I thought you were taking us to where these people live?" Before you can answer, some Remnant approach. "Welcome to our encampment, representatives." Chant the Remnant. "Captain <first> brought us your missive, and we asked them to bring you to meet us here. Would you care to come inside?" They gesture towards the largest of the tents. While obviously surprised, Ambassador Yashili aquiesces and the Hai step inside. Three of the Remnant accompany them, and wave for you to follow. The remaining Remnant continue patrolling around the camp.`
			`	The Remnant offer food and water to everyone before taking a seat on cushions on the ground. Looking around, Ambassador Yashili begins. "We come representing the Hai people, who live far from here. Some of our brothers and sisters have turned violent toward us, and have recently created a new ship that may cause us trouble. We have created a new ship of our own in retaliation, but we are still in need of technology capable of repairing the hull of a ship while in flight so that we may have an advantage. Captain <last> has told us that you have such technology, with ship hulls that repair themselves."`
			`	"This is true that our ships self repair," one of the Remnant responds. "But this is not technology that we are able to share. This property is intrinsic to the metals that we build our ships with, metals which are in low supply." Yashili looks concerned with this development; perhaps this journey was a worthless one.`
			choice
				`	"So is there no way that you can help them?"`
				`	"Alright, let's return to Hai-home."`
			`	"Wait. We may still have something that can help," a different Remnant chants. "For many decades, we have been fighting and plundering from Korath raiders from the Core. Their ships contain outfits capable of hull repair through the use of small robots. We have been unable to reproduce these robots with ease, but we may be able to provide for you our research and a number of samples that you may use to reproduce this technology."`
			`	"We too have encountered these Korath," Yashili says in response. "Although, we had not taken their technology. This may be of great use to us."`
			`	"We would like to ask something in trade, however. We would like a copy of your astronomical records. The history of the galaxy itself is a mystery to us, and we would like to learn more about it."`
			`	Ambassador Yashili looks thoughtful, then nods. "Historical astronomical data is scientifically valuable, as is learning how to repair our ships in flight. This seems a fair trade." She nods to one of her aides who begins tapping out commands on a small computer. "We should be able to have the data for you shortly. My assistant will need to use Captain <first>'s ftl communication system, but it shouldn't be long."`
			`	As the assistant scurries out, Ambassador Yashili and the Remnant settle into diplomatic small talk. The conversational dance between a group of professional diplomats and a secretive group that are currently playing the role of nomadic desert traders is an odd one.`

mission "Strider 4 (Remnant)"
	name "Technology Obtained"
	description "The delegation has successfully obtained hull repair technology from the Remnant. Return to <destination> so that the Hai can begin developing this technology for their new ship."
	landing
	source "Deadman's Cove"
	destination "Hai-home"
	passengers 3
	cargo "repair technology" 12
	blocked "You need <capacity> in order to return the Hai delegation and the repair technology back to <planet>."
	to offer
		has "Strider 3 (Remnant): done"
	on offer
		conversation
			`An hour later the ambassador's assistant returns with a large data drive, and hands it to Ambassador Yashili. The Remnant pull out a datachip of their own, and the two groups formally exchange the two drives. In response to an unseen signal, another Remnant pulls back one side of the tent and a team carries in a segment from a systems core with several of the small robots. "There is more of these waiting outside for you, but here is a sample for you to inspect. Using the instructions we have provided, you should have no problem making use of this technology." They poke at the console, and the robots promptly spring to life and assembled a small pile of parts into a model ship.`
			`	"Thank you for doing business with us." Says Ambassador Yashili. "You have a friend in the Hai." She bows, and then leads the delegation out of the tent to back to your ship. The Remnant return the bow with fluid grace, and quickly shoulder the Korath equipment. They quickly load everything onto the <ship> and you set out to return to <destination>.`
			`	A final glance at your exterior cameras shows the Remnant have already almost finished packing up their encampment onto camels, with a few of them carrying equipment that looks very much like portable scanning equipment. As one tent comes down, it reveals a anti-aircraft artillery canon installed on a sled. Despite the rustic appearance they portrayed, the Remnant certainly took their security seriously. There is still no ship visible, but you suspect that a starling or gull is undoubtably hiding nearby.`
				accept

```

Reviewer 2: And I just realized I never actually *finished* Strider 4 (Remnant). I'll do that later on today.

Reviewer 2: OK, realized that actually should be blank so that both it and the coalition string can have the completed dialog in the wrap-up conversation. Updated this suggestion with my changes and spelling corrections, and should be ready to commit.

			`	"That is most concerning," Yashili says with nods of agreement from the other two members of the delegation. "Hopefully the presence of the Quarg in Hai space will be of no issue to them." Yashili thinks for a moment. "Or perhaps we could use this information to our advantage."`
			label normal
			`	The delegation gathers their things and boards your ship, and you plot a course for <destination>. Hopefully the Coalition will be willing to share with an alien species they just met.`
				accept
	on visit
		dialog `It appears that the delegation is bunked on escorts that have not arrived in this system yet. Take off and wait for them.`
	on complete
		conversation
			`You notify the Heliarch that you're bringing alien ambassators to meet with them, and surely enough a delegation is waiting for you when you land. They briefly salute you, but appear to be much more interested in the strange new beings you've brought to their center of government.`
			`	The Hai delegation is momentarily confused by the unusual speech pattern the Coalition species have, but snap out of it shortly.`

> 			`	The Heliarch then start breaking it to the Hai their story, the same they told you when you first met them. While both delegations are engaged in conversation, a Heliarch agent approaches you, asking, "To this precious encounter, with what purpose have you brought these people, Captain?"`

Reviewer 3: ```suggestion
			`	The Heliarch then start explaining their story to the Hai, the same they told you when you first met them. While both delegations are engaged in conversation, a Heliarch agent approaches you, asking, "To this precious encounter, with what purpose have you brought these people, Captain?"`
```

			`	The others seem to be in an excited conversation of their own, after turning off their translation devices. Yashili and the other Hai also turn to you, looking somewhat confused, waiting for you to respond for them.`
			choice
				`	"The Quarg are indeed there, but they stay out of Hai business, and the Hai stay out of theirs."`
					goto nottheirbusiness
				`	"Yes, the Quarg have a single system there with a complete ringworld, where many Hai and even some humans also live."`
			`	The Heliarchs speak among themselves at a rapid pace, and before you know it several dozen other agents are around. One of them, an older Saryd with a golden circlet, says, "Tell us more about this ringworld, you must, Hai friends. Accompany us, would you?"`
			`	Without waiting for the Hai delegation to answer, they practically drag them to a restricted section of the ringworld. About an hour later, the Hai are brought back, most with tired expressions.`
			`	"Had I known we would be interrogated on the engineering aspects and the inner workings of a ringworld, I would have had an engineer come with us here," Yashili whispers to you.`
			
			label nottheirbusiness

> 			`	The Heliarchs once again warn the Hai to be careful with the Quarg, and not to trust them, before finally letting the Hai say the reason for their trip here.`

Reviewer 3: ```suggestion
			`	The Heliarchs once again warn the Hai to be careful with the Quarg before finally letting the Hai explain the reason for their trip here.`
```
Just personal preference. A little unsure about the commas surrounding `and not to trust them`, so I just removed the phrase to avoid the issue.

> 			`	The delegation still looks a bit overwhelmed, as if still processing all the history the Heliarch just told them, but Yashili has kept her composure and begins speaking. "We come representing the Hai people, who live far from here. Some of our brothers and sisters have turned violent toward us, and have recently created a new ship that may cause us trouble. We have created a new ship of our own in retaliation, but we are still in need of technology capable of repairing the hull of a ship while in flight so that we may have an advantage. Captain <last> has told us that you obtain such technology, with ship hulls that repair themselves."`

Reviewer 3: ```suggestion
			`	The delegation still looks a bit overwhelmed, as if still processing all the history the Heliarch just told them, but Yashili has kept her composure and begins speaking. "We come representing the Hai people, who live far from here. Some of our brothers and sisters have turned violent toward us, and have recently created a new ship that may cause us trouble. We have created a new ship of our own in retaliation, but we are still in need of technology capable of repairing the hull of a ship while in flight so that we may have an advantage. Captain <last> has told us that you possess such technology."`
```
This change is just copied and pasted from the one I suggested for the Remnant since the dialogue's the same.

			choice
				`	"So are you willing to give or sell one of those outfits to the Hai?"`
				`	"Would you share the way of mass producing those outfits with the Hai?"`
			`	"Much involved in projecting our repair modules, House Chamgar is," the same Arach says. "Contact their representatives in this ring, we will."`
			`	The Heliarch ask that the Hai prepare to describe their ships to the members of the great Arach House, who will then prepare a copy of the blueprints for the repair modules. You and the Hai delegation decide to meet up again in the spaceport in a few hours.`

mission "Strider 3 (Coalition)"
	name "Technology Obtained"
	description "The delegation has successfully obtained hull repair technology from the Coalition. Return to <destination> so that the Hai can begin developing this technology for their new ship."
	source "Ring of Friendship"
	destination "Hai-home"
	passengers 3
	cargo "repair technology" 12
	blocked "You need <capacity> in order to return the Hai delegation and the repair technology back to <planet>."
	to offer
		has "Strider 2 (Coalition): done"
	on offer
		conversation

> 			`The Hai are waiting for you, and confirm the Heliarchs handed them a data chip containing schematics for the hull repair modules and their hull-walking robots, as well as a and load a crate with two small hull repair modules onto your ship for the Hai. Yashili thanks the Coalition for their generosity and boards your ship, ready to return to <destination>.`

Reviewer 3: ```suggestion
			`The Hai are waiting for you, and confirm the Heliarchs handed them a data chip containing schematics for the hull repair modules and their hull-walking robots. Some workers load a crate with two small hull repair modules onto your ship for the Hai. Yashili thanks the Coalition for their generosity and boards your ship, ready to return to <destination>.`
```

			"Flea" 24
			"Lightning Bug" 2
			"Water Bug"
		add variant 3
			"Pond Strider" 3
			"Flea" 24
			"Lightning Bug (Pulse)" 2
		add variant 2
			"Pond Strider (Pulse)" 3
			"Flea" 24
			"Shield Beetle"
		add variant 2
			"Shield Beetle (Pulse)"
			"Pond Strider (Railgun)" 3
			"Flea" 24
		add variant 1
			"Pond Strider (Ion)" 3
			"Flea" 24
			"Shield Beetle"
			"Lightning Bug (Pulse)" 2
		add variant 1
			"Shield Beetle (Pulse)"
			"Pond Strider (Tracker)" 3
			"Flea" 24
			"Lightning Bug" 2
		add variant 1
			"Pond Strider (Tracker)" 3
			"Flea" 24
			"Centipede" 3
		add variant 2
			"Centipede"
			"Shield Beetle (Pulse)" 2
			"Pond Strider (Railgun)" 3
			"Flea" 24
		add variant 1
			"Geocoris"
			"Shield Beetle (Pulse)"
			"Pond Strider (Tracker)" 3
			"Flea" 24
		add variant 1
			"Geocoris"
			"Pond Strider (Pulse)" 3
			"Flea" 24
			"Grasshopper"
			"Grasshopper (Tracker)"
		add variant 1
			"Geocoris"
			"Pond Strider (Pulse)" 3
			"Flea" 24
			"Grasshopper"
			"Grasshopper (Tracker)"
		add variant 1
			"Geocoris (Tracker)"
			"Aphid (Anti-Missile)" 2
			"Aphid"
			"Pond Strider" 3
			"Flea" 24

mission "Strider 4"
	landing
	source "Hai-home"
	to offer
		or

> 			has "Strider 3 (Remnant): done"

Reviewer 2: ```suggestion
			has "Strider 4 (Remnant): done"
```

I used one extra mission on the Remnant side of things, so if the suggestion is merged instead of the PR to the base branch, then this suggestion will need to be merged to handle it properly.

> mission "Strider 4"
> 	landing
> 	source "Hai-home"
> 	to offer
> 		or
> 			has "Strider 3 (Remnant): done"
> 			has "Strider 3 (Coalition): done"
> 	on offer
> 		event "pond strider mass production" 30
> 		conversation
> 			``
> 				decline

Reviewer 2: This doesn't need to exist. Simply put the `event "pond strider mass production" 30` in the `on complete` section of `Strider 3 (Coalition)` and `Strider 4 (Remnant)`. It will be simpler, and doesn't require an empty conversation, either.

```suggestion
```

Reviewer 4: You misunderstand the purpose of this mission. The Coalition and Remnant branches are (or were) suppose to both converge on this final mission where the Hai make comment about the technology discovered and tell you when the new Pond Strider will be ready. Notice how the Coalition branch currently has nothing happen when you return to Hai space; that's what this mission is suppose to be (hence the empty conversation and me saying earlier that this isn't finished).

Reviewer 2: Ah, OK. That makes sense, then. I missed that comment somewhere. I'll delete the suggestion part of the comment to save some space and remove the distraction.

mission "Strider notification"
	invisible
	landing
	source
		government "Hai"
	to offer
		has "event: pond strider mass production"
	on offer

> 		log `For assisting the Hai establish an alien embassy, they've made newly developed warships and outfits available.`

Reviewer 2: While this log entry is good for the Coalition route, it doesn't match the Remnant route. Having separate log entries would be good, but alternatively slightly modifying it as follows would also work.

```suggestion
		log `For assisting the Hai conduct negotiations with an alien faction, they have made newly developed warships and outfits available.`
```

		conversation
			scene `thumbnail/hai pond strider`

> 			`The Hai thank you for your assistance with interspecies diplomacy. They have begun production of some new military ships and outfits that they've granted you access to in their shipyards and outfitters.`

Reviewer 3: ```suggestion
			`The Hai thank you for your assistance with interspecies diplomacy. They have begun production of some new military ships and outfits, and you'll be granted access to them in return for your help.`
```

			`	"Greetings, Captain <first>," The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction. It is to my understanding that they desire the means to repair the hull of a ship in flight. Is this correct?"`
			choice
				`	"Yes. The Hai are hoping to be able to improve a new carrier they are developing to repair its drones."`
				`	"Yes. The Hai want to be able to repair the hulls of their ships to gain an edge against their enemies."`
			`	"So the Hai want to improve their military technology? I presume this is in response to their long-standing war against the Unfettered?"`
			choice
				`	"Yes, although they don't think of it as such."`
				`	"It isn't a war; it's a defensive action."`
			`	Rayn looks amused. "It does not matter what they call it. A war by any name is just as bloody. But that aside, how do they know about us to make such a request?"`
			choice
				`	"They were looking for someone who had repair technology, and I said I could find someone."`
				`	"They asked if I knew anyone who had in-flight repair technology, and I said I knew a group of human researchers."`

> 			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes his eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these coordinates on Covert in the Gienah system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`

Reviewer 2: ```suggestion
			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes their eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these coordinates on Covert in the Gienah system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`
```

mission "Strider: Remnant 3"
	landing
	name "Technology Obtained"
	description "The delegation has successfully obtained hull repair technology from the Remnant. Return to <destination> so that the Hai can begin developing this technology for their new ship."
	landing
	source "Covert"
	destination "Hai-home"
	passengers 3
	cargo "repair technology" 12
	blocked "You need <capacity> in order to return the Hai delegation and the repair technology back to <planet>."
	to offer
		has "Strider: Remnant 2: done"
	on offer
		log `Helped the Hai get in contact with the Remnant in order to learn how to create in-flight hull repair technology. The Hai obtained a copy of the Remnant's research on Korath systems cores, along with a number of samples.`
		conversation
			`At the designated coordinates, there is a small encampment set up that looks vaguely reminiscent of a desert trader outpost. Aside from a few camels tied near a water trough, the place looks abandoned. When you touch down next to it, the delegation steps out cautiously. "Is this the right place?" one of the Hai asks somewhat worriedly.`

> 			`	Before you can answer, a number of Remnant emerge from the tents, and you notice that Rayn is among them. "Welcome to our encampment, representatives," Rayn chants. "Captain <first> brought us your missive, and we asked them to bring you here. Would you care to come inside?" He gestures toward the largest of the tents. While obviously surprised, Ambassador Yashili acquiesces, and the Hai step inside. Rayn gestures for you to follow. The remaining Remnant begin patrolling around the camp.`

Reviewer 2: ```suggestion
			`	Before you can answer, a number of Remnant emerge from the tents, and you notice that Rayn is among them. "Welcome to our encampment, representatives," Rayn chants. "Captain <first> brought us your missive, and we asked them to bring you here. Would you care to come inside?" They gestures toward the largest of the tents. While obviously surprised, Ambassador Yashili acquiesces, and the Hai step inside. Rayn gestures for you to follow. The remaining Remnant begin patrolling around the camp.`
```

			`	"Oh, yes," Osen says. "It may take us a few months to get everything working. A single month at the least if we are lucky."`
			`	"This is good to know. The sooner we have these ships function, the better."`
			`	Osen once again gets ready to run off, but this time stops himself and turns to you. "Thank you Captain. Without this information, it could have taken me years to finish this project." You nod in response, and Osen finally runs off to his shipyard.`
			`	"I would like to thank you as well," Yashili says. "It has been some time since I have had the chance to leave our space. It is always exciting to go to new places and meet new people." Yashili bows to you, and you return a bow of your own.`
			`	"And we also need to express our gratitude," one of the elders says. "It is good to have you as a friend of the Hai. We shall keep you no longer, though. May you stay safe on your adventures."`
				decline

mission "Strider notification"
	invisible
	landing
	source
		government "Hai"
	to offer
		has "event: pond strider mass production"
	on offer
		conversation
			scene `thumbnail/hai pond strider`

> 			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoiler the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`

Reviewer 5: The second last sentence should read:
I won't **spoil** the surprise for you though.

			choice
				`	(Yes.)`
					goto start
				`	(Not right now.)`
					defer
			
			label "know coalition"
			`You've returned to <planet> to help the Hai to develop hull repair technology. Torsa's father asked if you knew of any aliens with hull repair technology, and you have met the Coalition, who have outfits capable of repairs hulls. Would you like to tell the elders about the Coalition?`
			choice
				`	(Yes.)`
					goto start
				`	(Not right now.)`
					defer
			
			label "know remnant"
			`You've returned to <planet> to help the Hai to develop hull repair technology. Torsa's father asked if you knew of any aliens with hull repair technology, and you have met the Remnant, who have ships that repair themselves. Would you like to tell the elders about the Remnant?`
			choice
				`	(Yes.)`
				`	(Not right now.)`
					defer
			
			label start

> 			`	You inform the elders that you have discovered a group that could help them with creating hull repair technology and they invite you to the council chamber. "Greetings, Captain <last>," one of the elders says to you upon entering the chamber. "What news do you have to bring us on your search for hull repair technology?"`

Reviewer 3: ```suggestion
			`	You inform the elders that you have discovered a group that could help them with creating hull repair technology, and they invite you to the council chamber. "Greetings, Captain <last>," one of the elders says to you upon entering the chamber. "What news do you have to bring us on your search for hull repair technology?"`
```
All good if you don't want the word change here, but still need the comma or it's grammatically incorrect.

Reviewer 2: I can't accept the changes (I'm not the author), but I think it looks good.

	on complete
		conversation
			`Shortly after landing on <planet>, you spot Ambassador Yashili's aide rushing across the tarmac to meet you. "Please tell me they agreed. They did, right?" You barely have the chance to nod before the aide continues on. "The Ambassador will be ready in just a moment. She will meet you in the starport shortly. Thank you!" And with that, the officious Hai scurries off back the way it came.`

mission "Strider: Remnant 2"
	name "Contact the Remnant"
	description "Transport Ambassador Yashili and her delegation to <destination> to attempt to obtain hull repair technology from the Remnant."
	source "Hai-home"
	destination "Deadman's Cove"
	infiltrating
	passengers 3
	blocked "The Hai are sending a delegation to <destination> and will require 3 bunks for the trip."
	to offer
		has "Strider: Remnant 1: done"
	on offer
		conversation

> 			`You enter the spaceport and find the Hai delegation meant to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I heard that we will be travelling a great distance across the galaxy. May we have a safe journey."`

Reviewer 5: We already had the introduction to Yashili in `Strider: Remnant 1` so this is redundant.

			`	As the assistant scurries out, Ambassador Yashili and the Remnant settle into diplomatic small talk. The conversational dance between a group of professional diplomats and a secretive group that are currently playing the role of nomadic desert traders is an odd one.`

mission "Strider: Remnant 3"
	name "Technology Obtained"
	description "The delegation has successfully obtained hull repair technology from the Remnant. Return to <destination> so that the Hai can begin developing this technology for their new ship."
	landing
	source "Deadman's Cove"
	destination "Hai-home"
	passengers 3
	cargo "repair technology" 12
	blocked "You need <capacity> in order to return the Hai delegation and the repair technology back to <planet>."
	to offer
		has "Strider: Remnant 2: done"
	on offer
		conversation

> 			`An hour later, the ambassador's assistant returns with a large data drive and hands it to Ambassador Yashili. The Remnant pull out a data chip of their own, and the two groups formally exchange the data. In response to an unseen signal, another Remnant pulls back one side of the tent, and a team carries in a segment from a systems core with several of the small robots. "There is more of these waiting outside for you, but here is a sample for you to inspect. Using the instructions we have provided, you should have no problem making use of this technology." They poke at the console, and the robots promptly spring to life and assemble a small pile of parts into a model ship.`

Reviewer 5: Shouldn't it be `"There **are** more of these waiting outside for you`

					goto coalition
				`	"There is a group of humans far from here known as the Remnant who have ships that can repair themselves."`
					goto remnant
			
			label "tell remnant"
			choice
				`	"There is a group of humans far from here known as the Remnant who have ships that can repair themselves."`
					goto remnant
			
			label "tell coalition"
			choice
				`	"There is a group of three alien species far from here known as the Coalition that have outfits able to repair a ship in flight."`
					goto coalition
			
			label remnant
			apply
				set "strider: remnant"
			`	"A group of humans with hull repair technology?" one of the elders asks with a puzzled look. "I am surprised that we have not heard of these humans before. Are they well known among the Republic?"`
			`	"No," you explain. "They are a reclusive group unknown to most of humanity. They likely wouldn't take kindly to being visited by aliens."`
				goto end
			
			label coalition
			apply
				set "strider: coalition"
			`	"A group of three aliens?" one of the elders asks with a puzzled look. "It is not often in the galaxy that such groups appear."`
			
			label end

> 			`	"We will put together a delegation for you to transport to these people you mention. They will be waiting in the spaceport shortly," another elder says. "From there you can lead them how you see fit to make contact with this group. We trust that you will be capable in doing so."`

Reviewer 3: ```suggestion
			`	"We will put together a delegation for you to transport to these people you mention. They will be waiting in the spaceport shortly," another elder says. "You may lead them to make contact with this group how you see fit. We trust that you will be capable in doing so."`
```
This is optional. The phrasing of that second-to-last sentence is a little strange. Don't really know if this suggestion is better worse.

				decline

mission "Strider: Remnant 1"
	name "Contact the Remnant"
	description "Take a message from the Hai to <destination> to set up a meeting between the Remnant and the Hai."
	source "Hai-home"
	stopover
		government "Remnant"
	to offer
		has "strider: remnant"
	on offer
		conversation

> 			`You enter the spaceport and find a Hai delegation ready to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I wasn't sure how far we will be travelling, so I packed assuming we will be travelling a great distance across the galaxy."`

Reviewer 3: ```suggestion
			`You enter the spaceport and find a Hai delegation ready to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I'm not sure how far we will be travelling, so I packed assuming we will be going a great distance across the galaxy."`
```
OR
```suggestion
			`You enter the spaceport and find a Hai delegation ready to travel to the Remnant to negotiate for the hull repair technology. The leader of the delegation introduces herself as Ambassador Yashili. "I wasn't sure how far we would be travelling, so I packed assuming we will be going a great distance across the galaxy."`
```

			choice

> 				`	"Wait a minute! This group doesn't react well to strangers. How about I setup a meeting first?"`

Reviewer 3: ```suggestion
				`	"Wait a minute! This group doesn't react well to strangers. How about I set up a meeting first?"`
```
When it's a verb, `set up` is two words.

> 				`	"This is a bad idea. Sorry, but I am backing out."`

Reviewer 3: ```suggestion
				`	"This is a bad idea. Sorry, but I'm backing out."`
```

					decline

> 			`	The Hai looking at each other and chatter quietly among themselves, then Ambassador Yashili speaks up. "You know this group, and how to make the best impression on them." She pauses to whisper to another Hai, who runs off. The ambassador turns back to you. "I will provide you with a letter with a brief introduction to us, an explanation of why we would like to meet them, and an invitation to establish diplomatic relations."`

Reviewer 3: ```suggestion
			`	The Hai look at each other and chatter quietly among themselves, then Ambassador Yashili speaks up. "You know this group, and how to make the best impression on them." She pauses to whisper to another Hai, who runs off. The ambassador turns back to you. "I will provide you with a letter with a brief introduction to us, an explanation of why we would like to meet them, and an invitation to establish diplomatic relations."`
```

			`	Moments later, the second Hai runs back into the spaceport carrying a sealed envelope and hands it to the Ambassador. She turns to you. "Here it is. Please take this to these reclusive humans for us."`
				accept
	on stopover
		conversation

> 			`Looking around the spaceport, you are not sure who to talk to about such a request. Eventually you go to the starport information desk to ask them. After a few minutes they tell you that someone will meet with you, and invite you to wait in the cafeteria.`

Reviewer 3: ```suggestion
			`Looking around the spaceport, you are not sure who to talk to about such a request. Eventually, you go to the starport information desk to ask them. After a few minutes, they tell you that someone will meet with you and invite you to wait in the cafeteria.`
```

			`	After an hour of watching Remnant eating and gesturing to each other, an androgynous Remnant approaches your table. You try to decide what age they are, but while they do not seem young, neither do they seem particularly old.`
			branch signlanguage
				has "Remnant: Learn Sign Follow Up: offered"

> 			`	"Greetings, Captain <first>." The stranger chants. "My name is Rayn, and my colleague said you had a request from another faction?"`

Reviewer 3: ```suggestion
			`	"Greetings, Captain <first>." The stranger chants. "My name is Rayn, and my colleague said you had a request from another faction. How can I help?"`
```
That sentence isn't grammatically a question.

				goto stridercontinue
			label signlanguage

> 			`	"Greetings, Captain <first>." The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction?"`

Reviewer 3: ```suggestion
			`	"Greetings, Captain <first>." The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction. How can I help?"`
```

			label stridercontinue
			choice

> 				`	"The Hai are hoping to learn from someone."`

Reviewer 3: Is it established to the player before now that the Remnant 1) know the Hai, and 2) know enough about them that it's not surprising the Remnant immediately know about their conflict? I don't have the strongest grasp on the mission structure in relation to the other Remnant missions, so just thought it would be worth checking.

Reviewer 2: It may or may not be established with the player that the Remnant know about the Hai, however the two resulting situations that I see are:
a) That the player is surprised that the Remnant know about the Hai, and
b) That the player has had a few hints that the Remnant are very well travelled and probably know about the Hai.

I don't see a problem with either situation. Either they didn't know the Remnant knew about them, and this is their chance to find out; or they did, and this is just normal. Either way works.

Reviewer 3: Fair enough. I'd agree with your assessment.

				`	"The Hai need help defending their space."`
			`	"So the Hai want to improve their military technology? I presume this is in response to their long-standing war against the Unfettered?"`
			choice
				`	"Yes, although they don't think of it as such."`

> 				`	"It isn't a war, it is a defensive action."`

Reviewer 3: ```suggestion
				`	"It isn't a war; it's a defensive action."`
```

> 			`	Rayn looks amused. "It does not matter what they call it, a war by any name is just as bloody. But that aside, how do they know about us to make such a request?"`

Reviewer 3: ```suggestion
			`	Rayn looks amused. "It does not matter what they call it. A war by any name is just as bloody. But that aside, how do they know about us to make such a request?"`
```

			choice

> 				`	"They asked if I knew anyone who had in-flight repair technology, and I said I knew a group of human researchers."`

Reviewer 3: So is the player sort of forced into telling a lie/not the whole truth? This isn't exactly how `<first>` describes the Remnant in either case (at least from my own perspective).

Reviewer 2: The two main statements the player makes about the Remnant to the Hai are:
`	"There is a group of humans far from here known as the Remnant who have ships that can repair themselves."`

`	"No," you explain. "They are a reclusive group unknown to most of humanity. They likely wouldn't take kindly to being visited by aliens."`

While the second might be interpreted as implying they are external to mainstream humanity, but they could be just as much referring to some group hidden somewhere within humanity (because humanity seems to like having hordes of secretive groups, so not really surprising, right?)

Reviewer 3: I see where you're coming from, but I do think saying this directly to the Remnant without mentioning that the Hai now their name and that they're something of a secret still feels like misdirection to me.
It's up to you, though. It does get into subjective interpretation of the text, and you're head of Remnant stuff. If you think it's a non-issue, then you've got the final word.

Reviewer 5: When I first read it, my thoughts went straight to what @W1zrad was thinking – it feels like a misdirection in how it is said.

				`	"They were looking for someone who had repair technology, and I said I could find someone."`

> 			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes his eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these co-ordinates on Deadman's Cove, in the Almach system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`

Reviewer 3: ```suggestion
			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes his eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these co-ordinates on Deadman's Cove in the Almach system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`
```
The rule of thumb in-game seems to be no comma between planet and system, according to the `<destination>` phrases.

	on complete
		conversation

> 			`Shortly after landing on <destination> you spot Ambassador Yashili's aide rushing across the tarmac to meet you. "Please tell me they agreed? They did, right?" You barely have the chance to nod before the aide continues on. "The Ambassador will be ready in just a moment. She will meet you in the starport shortly. Thank you!" And with that, the officious Hai scurries off back the way it came.`

Reviewer 3: ```suggestion
			`Shortly after landing on <planet>, you spot Ambassador Yashili's aide rushing across the tarmac to meet you. "Please tell me they agreed. They did, right?" You barely have the chance to nod before the aide continues on. "The Ambassador will be ready in just a moment. She will meet you in the starport shortly. Thank you!" And with that, the officious Hai scurries off back the way it came.`
```
Same here with the sentence not being a question.

			`	"Greetings, Captain <first>," The stranger signs. "My name is Rayn, and my colleague said you had a request from another faction. It is to my understanding that they desire the means to repair the hull of a ship in flight. Is this correct?"`
			choice
				`	"Yes. The Hai are hoping to be able to improve a new carrier they are developing to repair its drones."`
				`	"Yes. The Hai want to be able to repair the hulls of their ships to gain an edge against their enemies."`
			`	"So the Hai want to improve their military technology? I presume this is in response to their long-standing war against the Unfettered?"`
			choice
				`	"Yes, although they don't think of it as such."`
				`	"It isn't a war; it's a defensive action."`
			`	Rayn looks amused. "It does not matter what they call it. A war by any name is just as bloody. But that aside, how do they know about us to make such a request?"`
			choice
				`	"They were looking for someone who had repair technology, and I said I could find someone."`
				`	"They asked if I knew anyone who had in-flight repair technology, and I said I knew a group of human researchers."`

> 			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes his eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these coordinates on Covert in the Gienah system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`

Reviewer 2: 👍 Being the one who wrote that section, I'd go with suggestion 1.

> 			`	"So they do not actually know about us. That is good." Rayn looks thoughtful and closes his eyes for a moment, then continues. "Pick up the Hai diplomats, but do not tell them where you are going, and take them to these coordinates on Covert in the Gienah system. It is a location in an uninhabited area where we will be able to talk. We will be waiting with something that might be able to help them - if they can meet our price."`

Reviewer 2: Good catch. adjusting this slightly to continue this would be good.

		dialog `It appears that the delegation is bunked on escorts that have not arrived in this system yet. Take off and wait for them.`
	on complete
		conversation

> 			`At the designated co-ordinates there is a small encampment set up, looking vaguely reminiscent of desert traders from old movies. You note there are even a few camels tied near a water trough. When you touch down next to it, the delegation steps out with a confused look. "Captain, I thought you were taking us to where these people live?" Before you can answer, some Remnant approach. "Welcome to our encampment, representatives." Chant the Remnant. "Captain <first> brought us your missive, and we asked them to bring you to meet us here. Would you care to come inside?" They gesture towards the largest of the tents. While obviously surprised, Ambassador Yashili acquiesces and the Hai step inside. Three of the Remnant accompany them, and wave for you to follow. The remaining Remnant continue patrolling around the camp.`

Reviewer 3: ```suggestion
			`At the designated co-ordinates, there is a small encampment set up that looks vaguely reminiscent of desert traders from old movies. You note there are even a few camels tied near a water trough. When you touch down next to it, the delegation steps out with a confused look. "Captain, I thought you were taking us to where these people live. Is this the right place?" Before you can answer, some Remnant approach. "Welcome to our encampment, representatives," chant the Remnant. "Captain <first> brought us your missive, and we asked them to bring you to meet us here. Would you care to come inside?" They gesture toward the largest of the tents. While obviously surprised, Ambassador Yashili acquiesces, and the Hai step inside. Three of the Remnant accompany them and gesture for you to follow. The remaining Remnant continue patrolling around the camp.`
```
One sentence isn't a question, so I added a question at the end.
Some general grammar fixes.

			`	"This is true that our ships self repair," one of the Remnant responds. "But this is not technology that we are able to share. This property is intrinsic to the metals that we build our ships with, metals which are in low supply." Yashili looks concerned with this development; perhaps this journey was a worthless one.`
			choice
				`	"So is there no way that you can help them?"`
				`	"Alright, let's return to Hai-home."`
			`	"Wait. We may still have something that can help," a different Remnant chants. "For many decades, we have been fighting and plundering from Korath raiders from the Core. Their ships contain outfits capable of hull repair through the use of small robots. We have been unable to reproduce these robots with ease, but we may be able to provide for you our research and a number of samples that you may use to reproduce this technology."`
			`	"We too have encountered these Korath," Yashili says in response. "Although, we had not taken their technology. This may be of great use to us."`

> 			`	"We would like to ask something in trade, however. We would like a copy of your astronomical records. The history of the galaxy itself is a mystery to us, and we would like to learn more about it."`

Reviewer 3: ```suggestion
			`	"We would like to ask something in trade, however: a copy of your astronomical records. The history of the galaxy itself is a mystery to us, and we would like to learn more about it."`
```
Just removes the double usage of "We would like".

> 			`	Ambassador Yashili looks thoughtful, then nods. "Historical astronomical data is scientifically valuable, as is learning how to repair our ships in flight. This seems a fair trade." She nods to one of her aides who begins tapping out commands on a small computer. "We should be able to have the data for you shortly. My assistant will need to use Captain <first>'s ftl communication system, but it shouldn't be long."`

Reviewer 3: ```suggestion
			`	Ambassador Yashili looks thoughtful, then nods. "Historical astronomical data is scientifically valuable, as is learning how to repair our ships in flight. This seems a fair trade." She nods to one of her aides who begins tapping out commands on a small computer. "We should be able to have the data for you shortly. My assistant will need to use Captain <first>'s FTL communication system, but it shouldn't be long."`
```

			`	As the assistant scurries out, Ambassador Yashili and the Remnant settle into diplomatic small talk. The conversational dance between a group of professional diplomats and a secretive group that are currently playing the role of nomadic desert traders is an odd one.`

mission "Strider: Remnant 3"
	name "Technology Obtained"
	description "The delegation has successfully obtained hull repair technology from the Remnant. Return to <destination> so that the Hai can begin developing this technology for their new ship."
	landing
	source "Deadman's Cove"
	destination "Hai-home"
	passengers 3
	cargo "repair technology" 12
	blocked "You need <capacity> in order to return the Hai delegation and the repair technology back to <planet>."
	to offer
		has "Strider: Remnant 2: done"
	on offer
		conversation

> 			`An hour later the ambassador's assistant returns with a large data drive, and hands it to Ambassador Yashili. The Remnant pull out a datachip of their own, and the two groups formally exchange the data. In response to an unseen signal, another Remnant pulls back one side of the tent and a team carries in a segment from a systems core with several of the small robots. "There is more of these waiting outside for you, but here is a sample for you to inspect. Using the instructions we have provided, you should have no problem making use of this technology." They poke at the console, and the robots promptly spring to life and assembled a small pile of parts into a model ship.`

Reviewer 3: ```suggestion
			`An hour later, the ambassador's assistant returns with a large data drive and hands it to Ambassador Yashili. The Remnant pull out a data chip of their own, and the two groups formally exchange the data. In response to an unseen signal, another Remnant pulls back one side of the tent, and a team carries in a segment from a systems core with several of the small robots. "There is more of these waiting outside for you, but here is a sample for you to inspect. Using the instructions we have provided, you should have no problem making use of this technology." They poke at the console, and the robots promptly spring to life and assemble a small pile of parts into a model ship.`
```
A couple spelling/grammar corrections. One thing of note: in-game files says "data cards", so I don't know if the space applies to "data chips" as well. I've added it to my suggestion so the question is at least asked.
Edit: Later on, you use "data chip", so I'll assume this correction is correct.

> 			`	"Thank you for doing business with us." Says Ambassador Yashili. "You have a friend in the Hai." She bows, and then leads the delegation out of the tent to back to your ship. The Remnant return the bow with fluid grace, and quickly shoulder the Korath equipment. They quickly load everything onto the <ship> and you set out to return to <destination>.`

Reviewer 3: ```suggestion
			`	"Thank you for doing business with us," says Ambassador Yashili. "You have a friend in the Hai." She bows, and then leads the delegation out of the tent to back to your ship. The Remnant return the bow with fluid grace, and quickly shoulder the Korath equipment. They load everything onto the <ship>, and you set out to return to <destination>.`
```

> 			`	As you take off, a final glance at your exterior cameras shows the Remnant have already almost finished packing up their encampment onto camels, with a few of them carrying equipment that looks very much like portable scanning equipment. As one tent comes down, it reveals a anti-aircraft artillery canon installed on a sled. Despite the rustic appearance they portrayed, the Remnant certainly took their security seriously. There is still no ship visible, but you suspect that a starling or gull is undoubtedly hiding nearby.`

Reviewer 3: ```suggestion
			`	As you take off, a final glance at your exterior cameras shows the Remnant have already almost finished packing up their encampment onto camels. A few of them carry equipment that looks very much like portable scanners. As one tent comes down, it reveals a anti-aircraft artillery canon installed on a sled. Despite the rustic appearance they portrayed, the Remnant certainly took their security seriously. There is still no ship visible, but you suspect that a Starling or Gull is undoubtedly hiding nearby.`
```
Ship names are supposed to be capitalized, right?

			`	The delegation still looks a bit overwhelmed, as if still processing all the history the Heliarch just told them, but Yashili has kept her composure and begins speaking. "We come representing the Hai people, who live far from here. Some of our brothers and sisters have turned violent toward us, and have recently created a new ship that may cause us trouble. We have created a ship of our own in retaliation, but we are still in need of technology capable of repairing the hull of a ship while in flight so that we may have an advantage. Captain <last> has told us that you possess such technology."`
			`	"Developed such technology, indeed we have," an Arach agent responds. "Composed of several dozens of small repair bots, and their center of command, such outfits are."`
			choice
				`	"So are you willing to give or sell one of those outfits to the Hai?"`
				`	"Would you share the way of mass producing those outfits with the Hai?"`
			`	"Much involved in projecting our repair modules, House Chamgar is," the same Arach says. "Contact their representatives in this ring, we will."`

> 			`	A short time later, a group of Arachi representing House Chamgar arrive. They have the Hai describe their ships, using this information to optimize the repair module for Hai ships. They then prepare a copy of the blueprints, handing a data chip containing the schematics for the hull repair modules and their hull-walking robots to the Hai. Some workers then load a crate with two small modules onto your ship for the Hai to use as examples. Yashili thanks the Coalition for their generosity and boards your ship, ready to return to <destination>.`

Reviewer 3: So the Remnant make a trade for the technology. How come the Coalition don't? You could have the Hai give them something in return or write a line or two about why the Coalition are willing to give it away for free.

	to offer
		has "event: pond strider mass production"
	on offer
		conversation
			scene `thumbnail/hai pond strider`

> 			`Upon landing on <planet>, you receive a message from Torsa. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design, and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoiler the surprise for you, though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`

Reviewer 3: ```suggestion
			`Upon landing on <planet>, you receive a message from Torsa. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoiler the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
```
Both the commas in the extra-long sentence I removed one from are optional. I removed the one I did because I think it helps a reader interpret the structure and phrasing a bit better.
